### ISSUE TITLE
Silktide Accessibility Checker - role="option" causes the No accessible text was found "Option"

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7955]
T17249 - WCAG related question about checkbox
https://surveyjs.answerdesk.io/internal/ticket/details/T17249
***
v1.9.133
[View Plunker](https://plnkr.co/edit/Uo1eYq6eMfKlCTfZ)

With Silktide Accessibility Checker, the Screen Reader mode, the No accessible text was found "Option" accessibility issue appears when selecting/deselecting a checkbox option.
![image](https://github.com/surveyjs/survey-library/assets/22372972/e0223c96-dfe8-468f-ab72-6a012a0527f9)

It doesn't happen in v1.9.109: [View Demo](https://plnkr.co/edit/fgWPypw4z9doXAWV)

**Comment by JaneSjs** [comment]
We researched this issue. It appears that the Silktide Accessibility Checker uses a proprietary screen reader simulator. Unlike other screen readers such as the NVDA screen reader, Silktide's screen reader simulator is unable to read a check item's caption for some reason. This behavior is likely specific to the Silktide's screen reader simulator as checkbox items can be properly read by other screen readers.
